src/In.ProjectEKA.HipService/wwwroot/swagger.yaml

```diff
@@ -221,6 +222,69 @@ paths:
             application/xml:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+
+  /health-information:
+    post:
```

Comment:
Don't we have to authorize the Api?

Reply:
There is a consentmanager id in middleware which will be authorized by JWT or Mutual authentication